packages/tailwind-config/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@repo/tailwind-config",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "lint": "eslint . --max-warnings 0"
+  },
+  "exports": {
+    "./postcss-config": "./postcss.config.js",
+    "./tw-config": "./tailwind.config.ts",
+    "./css": "./global.css"
+  },
+  "devDependencies": {
+    "tailwindcss": "^3.4.1",
+    "postcss": "^8.4.33",
+    "autoprefixer": "^10.4.17",
+    "typescript": "^5.3.3"
```

Comment:
Why do we need this?

Reply:
Please check other dependencies and ensure if it is really needed for the project.

Reply:
I checked with the projects using it and found that it works only if tailwind is installed. But not sure if it's required in the config package. I'll test it out update the status here

Reply:
The intuition was correct and I did not need the node-modules in tailwind-config package. 

Except for the tailwindcss. I needed it to get the type of config file which is there in tailwindcss npm. Is it worth deleting tailwindcss from the config package and lose type safety for the config.ts file

Reply:
So we are adding in only the tailwindcss dependency for the config package